### ISSUE TITLE
Highlighting from Find function sticks

### ISSUE THREAD

**Issue by igrad** [atom-minimap/minimap#628]
Steps to reproduce:
- Say your have two panes open: one of the left and one on the right, both with files open (I noticed this while coding in Python).
- Hit `ctrl + f` to open the finder, specifically a string that's present in the right-hand panel and notice that the strings are highlight in the minimap on the right.
- Click in to the left-hand panel to shift focus
- Press Escape to exit the finder and notice that the highlighting in the minimap on the right disappears
- Click back into the right-hand panel to focus it and the minimap highlights the strings again
- Clicking back into the left-hand panel will also show any syntax highlighted there

Long story short: closing the finder does not block minimap from highlighting syntax like the finder does.

Minimap v4.29.6
Atom v1.19.3
